### PR TITLE
Generate routes.ts for the app!

### PR DESCRIPTION
**Pull request by wejendorp**

### CODE REVIEW

File: templates/kapeta/block-type-frontend/browser-routes.tsx.hbs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//#FILENAME:src/browser/routes.tsx:create-only
```
Comment (hofmeister) [inline templates/kapeta/block-type-frontend/browser-routes.tsx.hbs:1]: Make it "write-never" if it's only intended for the AI

File: templates/kapeta/block-type-frontend/browser-routes.tsx.hbs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Import the screen components and add routes for each. For example:
+
+import { LoginScreen } from './screens/LoginScreen';
+import { SignupScreen } from './screens/SignupScreen';
+import { DashboardScreen } from './screens/DashboardScreen';
+import { ApplicationScreen } from './screens/ApplicationScreen';
+
+Implement the routes as an array of objects for react-router-dom data routers, e.g. below:
+{{/ai-comment}}
+export const routes = [
```
Comment (hofmeister) [inline templates/kapeta/block-type-frontend/browser-routes.tsx.hbs:16]: Hardcoded?

File: templates/kapeta/block-type-frontend/browser-routes.tsx.hbs (hunk 1 above)
Comment (wejendorp) [reply templates/kapeta/block-type-frontend/browser-routes.tsx.hbs:1]: I feel like there's a concept missing - right now we combine write-never and `ai-type` into special handling.
How about a new file mode that works similar to `{{#ai-context}}` ? `:ai-only`?

Comment (hofmeister) [reply templates/kapeta/block-type-frontend/browser-routes.tsx.hbs]: But we probably dont want to generate this file in "normal" mode? That's why we use "write-never"

Comment (hofmeister) [reply templates/kapeta/block-type-frontend/browser-routes.tsx.hbs]: But yes we could improve our "ai-type" so that it's not written out in the template

**Pull request merged**